pkg/agent/agent_windows.go

```diff
@@ -70,7 +70,12 @@ func (i *Initializer) prepareHostNetwork() error {
 	i.nodeConfig.UplinkNetConfig.Index = adapter.Index
```

Comment:
A basic question - so in-band IPs must be on the uplink interface (before OVS bridge is configured)? How we make sure that (in OVS bridge creation)?

Reply:
So, you meant we will use the inbandInterface to be the uplink when creating the HNSNetwork if inbandInterface is specified? Could we add a comment somewhere about this?